test/test-examples-server-custom-bindings.js

```diff
@@ -0,0 +1,42 @@
+/*jshint expr: true*/
+"use strict";
+var chai = require("chai");
+var expect = chai.expect;
+chai.config.includeStack = true;
+var request = require("supertest-as-promised");
+var alexaAppServer = require("../index");
+
+describe("Alexa App Server with Examples & Custon Server Bindings", function() {
+  var testServer;
+
+  afterEach(function() {
+    testServer.stop();
+  });
+
+  it("mounts hello world app (HTTP only)", function() {
+      testServer = alexaAppServer.start({
+        port: 3000,
+        host_address: "127.0.0.1",
```

Comment:
I was going to test it via referring to the actual address (like `http://127.0.0.1:3000` and `https://127.0.0.1:6000`) the express server is hosting to, since it's not possible to check for the IP from the express instance.